playwright/lib/testUtils.ts

```diff
@@ -0,0 +1,18 @@
+import { Page } from "@playwright/test";
+
+export async function shareGoalPrivately(userOnePage: Page) {
+  await userOnePage.locator(".goal-tile").first().click();
+  await userOnePage
+    .locator("div")
```

Comment:
@vinaybadgujar102 can you use **.goal-dropdown** instead of location the goal circle like this?